content/docs/errors/file-in-mutpliple-collections.md

```diff
@@ -0,0 +1,81 @@
+---
+title: 'File is Multiple Collections Error'
+---
+
+This error occurs when a file is in multiple collections. This is usually causes by overlapping paths in in your collections or an error in a [match property](/docs/reference/collections#matchinclude).
+
+The common fix for this is updating or adding the [match property](/docs/reference/collections#matchinclude) to exclude or include certain files.
+
+## Example with exclude
+
+```ts
+// tina/schema.ts
+export default defineConfig({
+  collections: [
+    {
+      label: 'Blog Posts',
+      name: 'posts',
+      path: 'content/posts',
+      // ..
+    },
+    {
+      label: 'Featured Posts',
+      name: 'featuredPosts',
+      path: 'content/posts/featured',
+      // ...
+    },
+    //..
+  ],
+  // ...
+})
+```
+
+This would cause an overlap because the Blog Post collection will contain all of teh files in the Featured Posts collection. This can be fixed by changing update the [`match.exclude` property](/docs/reference/collections#matchexclude) on the Blog Posts collection to exclude all of the files in the Features Posts collection.
```

Comment:
```suggestion
The example configuration above would cause an overlap because the Blog Post collection will contain all of the files in the Featured Posts collection. This can be fixed by updating the [`match.exclude` property](/docs/reference/collections#matchexclude) on the Blog Posts collection to exclude all of the files in the Features Posts collection.
```